src/main/resources/sim_api.h

```diff
@@ -125,10 +127,17 @@ template <class T> class sim_api_t {
   }
   void init_channels() {
     pid_t pid = getpid();
+    char cwd[PATH_MAX];
+    if (getcwd(cwd, sizeof(cwd)) != NULL) {
+       printf("Current working dir: %s\n", cwd);
```

Comment:
Adding a printf here makes me nervous, do you need it?

Reply:
not really needed indeed, i will remove this line